app/components/agenda/agendaitem/agendaitem-case/subcase-titles-edit/component.js

```diff
@@ -38,7 +38,20 @@ export default class SubcaseTitlesEdit extends Component {
   @action
   async saveChanges() {
     set(this, 'isLoading', true);
-    await saveSubcaseTitles(get(this, 'item'), get(this, 'propertiesToSet'), get(this, 'propertiesToSet'), true);
+
+    const propertiesToSetOnAgendaitem = {
+      'title': this.get('title'),
+      'shortTitle': this.get('shortTitle'),
+      'showInNewsletter': this.get('showInNewsletter')
+    };
+    const propertiesToSetOnSubcase = {
+      'title': this.get('title'),
+      'shortTitle': this.get('shortTitle'),
+      'accessLevel': this.get('accessLevel'),
+      'confidential': this.get('confidential'),
```

Comment:
Hier twijfel ik wat aan of dit gaat werken zoals je verwacht.

Als je model agendaitem is, en je bouwt de `propertiesToSetOnSubcase`
Dan ga je volgens mij voorhebben dat accesslevel en confidential op null of undefined gaan gezet worden
Moet ik eens lokaal aftesten